server/app/controllers/ShippingController.ts

```diff
@@ -0,0 +1,95 @@
+import TYPES from '../constants/types';
+import { ShippingService } from '../services/ShippingService';
+import { inject } from 'inversify';
+import {
+  controller,
+  httpDelete,
+  httpGet,
+  httpPatch,
+  httpPost,
+  results,
+} from 'inversify-express-utils';
+import { BaseController } from './BaseController';
+import { Doc } from 'inversify-express-doc';
+import { IShipping } from '../models/ShippingModel';
+import { Request } from 'express';
+
+@controller('/shipping')
+export class ShippingController extends BaseController {
+  constructor(@inject(TYPES.ShippingService) private shippingService: ShippingService) {
+    super();
+  }
+
+  @Doc('Get Shipping List')
+  /*
+   * @desc        Get all shipments
+   * @route       GET /shipping
+   * @access      Public
+   * @returns     List shippings Json Format
+   */
+  @httpGet('/')
+  public async get(): Promise<results.JsonResult> {
+    try {
+      const shippings: IShipping[] = await this.shippingService.getShippingList();
+      return this.json(shippings);
+    } catch (err) {
+      return this.handleError(err);
```

Comment:
Small nitpick: could you try to get 100% coverage? This line (line 36) and line 54 aren't covered, you can check the CI.